src/util.ts

```diff
@@ -34,14 +36,14 @@ export interface IMakeUploadFileData {
 }
 
 export class Util {
-	// tslint:disable-next-line no-any
-	public static async DownloadFile(url: string, options: any = {}): Promise<Buffer> {
+	// tslint:disable-next-line:max-line-length
+	public static async DownloadFile<T extends Response | Response["body"]>(url: string, options: OptionsOfDefaultResponseBody = {}): Promise<Buffer> {
```

Comment:
What did you make this a template for? Some protocol implementations already depend on this method....

Reply:
The template is nowhere used in the method and a `Buffer` is always used. Sounds like the template isn't needed.

Reply:
I used that from the return type that got had but forgot to remove that generic parameter again. (first had it return Response)